security_monkey/constants.py

```diff
@@ -20,6 +20,18 @@
 
 """
 
+from flask import Flask
+from flask import render_template
+from flask.helpers import make_response
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import LoginManager
+import os
+
+app = Flask(__name__, static_url_path='/static')
```

Comment:
Lines 23-30:
```python
from flask import Flask
from flask import render_template
from flask.helpers import make_response
from flask_sqlalchemy import SQLAlchemy
from flask_login import LoginManager
import os

app = Flask(__name__, static_url_path='/static')
```

Could you replace all of these with:

```python
from security_monkey import app
```

Reply:
Sure. No problem.